src/Control/Monad/Linear.hs

```diff
@@ -148,7 +149,7 @@ get :: (Applicative m, Dupable s) => StateT s m s
 get = state dup
 
 put :: (Applicative m, Consumable s) => s %1-> StateT s m ()
-put = void . replace
+put = Data.void . replace
```

Comment:
It's completely unrelated, but do we have in our README, a documentation of our recommended qualifying conventions? That `Data.Functor.Linear` is typically qualified as `Data`?

Reply:
No, I"ll add this in a later PR.